recipe/build.sh

```diff
@@ -75,6 +75,9 @@ if [[ $(uname) == "Linux" ]]; then
       SKIPS+=(-skip); SKIPS+=(qttools)
       SKIPS+=(-skip); SKIPS+=(qtlocation)
       SKIPS+=(-skip); SKIPS+=(qt3d)
+      EXTRA_OPTIONS=""
+    else
+      EXTRA_OPTIONS="-gstreamer 1.0"
```

Comment:
Do you want to add a comment why we can't use gstreamer for osx-arm64?

Reply:
um, this is more for `MINIMAL_BUILD` option above. if all these extensions are skipped, then the build process complains that `-gstreamer` isn't a build option.

I'm not sure about OSX+gstreamer. It probably could work. I just didn't focus on that aspect.